src/main/java/org/example/interfaces/Identifiable.java

```diff
@@ -0,0 +1,6 @@
+package org.example.interfaces;
+
+public interface Identifiable {
+    int getId();
```

Comment:
it's seems like getter and setter - probably we need also a field - wdyt?

Reply:
yes, it is better to choose an abstract class as a solution to the issue because there is no need to modify methods after initialization. Also, abstract classes can contain variables without obligatory initialization unlike interfaces